README.MD

```diff
@@ -27,14 +32,15 @@ note on changes:
 
 ## Wishlist for changes / new features
 
+- on most projects we only use one module for data, and one for domain. We could simplify the
```

Comment:
Having the wishlist in the README.md is discovererable, but also very rigid, since edits require approval. 

I would prefer these to be converted to Issues with the label `improvement` or `feature` applied.

Also, I feel slightly anxious when picking up one of these, for fear that someone else is working on it. I would prefer to claim an issue by self-assignment

Reply:
great idea! I've added this wish to the wishlist :)